pkg/sosreport/sosreport.jsx

```diff
@@ -177,7 +177,8 @@ function sosCreate(args, setProgress, setError, setErrorDetail) {
     });
 
     task.catch(error => {
-        setError(error.toString());
+        console.error(JSON.stringify(error));
```

Comment:
Matter of time until someone drops this as forgotten debugging info :) Comment would be useful why it is there

Reply:
Added a comment. I also fixed the commit message for the fuser patch, that was misleading.